src/Authenticator/LoginHandler.php

```diff
@@ -457,6 +457,9 @@ public function redirectAfterSuccessfulLogin(): HTTPResponse
             return $this->redirect($this->getBackURL() ?: $loginUrl);
         }
 
+        // Redirecting after successful login expects a getVar to be set, store it before clearing the session data
+        $request['BackURL'] = $this->getBackURL();
```

Comment:
Do you think a comment might be neat for people that are confused about array access on objects?